### ISSUE TITLE
Website not loading additional layers (Linux-Server)

### ISSUE THREAD

**Issue by surcoufx83** [StrangeLoopGames/EcoIssues#7732]
Hi there,

I'm running my Server on Linux and since 0.6.2.5-beta, the map in the World Status page can't display additional layers (like AirPollutionSpread). Tested with Chrome (v 66) and Edge Browser

How to reproduce: Open the website and wait for map loaded -> Click on Select map layer and select something (happens for all layers) -> Map get's a little bit darker but nothing else changes (but error occurs in Chrome Browser console). 

Server versions: Mono linux standalone with 0.6.2.5, 0.7.3.3, 0.7.4.2
Edge Browser is not displaying ocean

Log from Chrome console when switching to AirPollutionSpread layer:
```
ecomap.js:419 Starting to parse AirPollutionSpread
ecomap.js:112 Uncaught TypeError: Cannot read property 'concat' of undefined
    at lzwDecode (ecomap.js:112)
    at parseImg (ecomap.js:311)
    at parseBlock (ecomap.js:331)
```

Default view without selected layer:
![2018-05-12 12_07_53-eco-worldmap](https://user-images.githubusercontent.com/1303974/39956322-510c3374-55de-11e8-8d8d-9aa46b9db1fe.png)

View with selected layer:
![2018-05-12 12_15_58-eco-layerselected](https://user-images.githubusercontent.com/1303974/39956323-51248348-55de-11e8-855c-66ece023dd3a.png)

Default view in Microsoft Edge:
![2018-05-12 12_16_11-microsoft edge](https://user-images.githubusercontent.com/1303974/39956324-513ff146-55de-11e8-9065-833a50563d5f.png)

**Comment by headswe** [comment]
![image](https://user-images.githubusercontent.com/425212/40266436-5bef0e02-5b4b-11e8-84ac-f50cccdf7762.png)

Pretty sure this issue is a problem with the generation of the gif for the layers, the end points for the layers just returns a sqaure.

**Comment by SeanAtr0n** [comment]
Fixed in 7.5
